src/shed/_codemods.py

```diff
@@ -295,6 +295,21 @@ def replace_unnecessary_nested_calls(self, _, updated_node):
 
         Unnecessary <list/reversed/sorted/tuple> call within <list/set/sorted/tuple>()..
         """
+        # If either of two nested sorted calls have a key, it's incorrect to try
+        # to merge them. Theoretically the keys could be combined into a tuple,
+        # but this is hard to make work in generality, and it's better to just
+        # leave this alone and let a human deal with it if they care.
+        if (
+            updated_node.func.value == "sorted"
+            and updated_node.args[0].value.func.value == "sorted"
+            and any(
+                arg.keyword and arg.keyword.value == "key"
```

Comment:
```suggestion
                arg.keyword and arg.keyword.value in ("key", "reverse")
```

I think we should also fix #80 and might as well do so here.

Reply:
Agreed - merging this as a safe improvement on status quo; we can discuss #80 on that issue.